R/queue.R

```diff
@@ -0,0 +1,78 @@
+#' Object for managing running jobs on the redis queue
+#'
+#' @keywords internal
+Queue <- R6::R6Class("Queue", #nolint
+  cloneable = FALSE,
+  public = list(
+    #' @field root Orderly root
+    root = NULL,
+    #' @field config Orderly config
+    config = NULL,
+
+    #' @description
+    #' Create object, read configuration and setup Redis connection.
+    #'
+    #' @param root Orderly root.
+    #' @param queue_id ID of an existing queue to connect to, creates a new one
+    #'   if NULL (default NULL)
+    #' @param workers Number of workers to spawn (default 1)
+    #' @param cleanup_on_exit If TRUE workers are killed on exit (defaults
+    #'   to TRUE if workers is > 0)
+    initialize = function(root, queue_id = NULL, workers = 1,
```

Comment:
I don't believe you want to do this, but if you *do* opt to retain it, rename arg to `n_workers` I think